tools/linux/uio_drivers/blk/blk.c

```diff
@@ -16,10 +16,16 @@
 #include <linux/fs.h>
 
 #include <sddf/blk/queue.h>
+#include <blk_config.h>
 
 #include <uio/libuio.h>
 #include <uio/blk.h>
 
+// @ericc:
+// This linux userspace block driver does not use DMA to transfer data between the device and the rest of the system.
+// As such we'll need to expose the data region to the driver for it to read/write data to/from the device.
+// Once we figure out how to do DMA in userspace (and get it working), we can remove this.
```

Comment:
Make this an issue instead of a todo. It is non-trivial.

Reply:
#57 listed as issue here